File: TransCoder_tokenized_test_set_functions/cpp/FIND_DIMENSIONS_RIGHT_ANGLED_TRIANGLE.cpp
void findDimen ( int H, int A ) {
  if ( H * H < 4 * A ) {
    cout << "Not Possible\n";
    return;
  }
  double apb = sqrt ( H * H + 4 * A );
  double asb = sqrt ( H * H - 4 * A );
  cout . precision ( 2 );
  cout << "P = " << fixed << ( apb - asb ) / 2.0 << "\n";
  cout << "B = " << ( apb + asb ) / 2.0;
}

File: TransCoder_tokenized_test_set_functions/python/FIND_DIMENSIONS_RIGHT_ANGLED_TRIANGLE.py
def findDimen ( H, A ) :
    if H * H < 4 * A :
        print ( "Not Possible" )
        return
    apb = sqrt ( H * H + 4 * A )
    asb = sqrt ( H * H - 4 * A )
    print ( "P = ", "%.2f" % ( ( apb - asb ) / 2.0 ) )
    print ( "B = ", "%.2f" % ( ( apb + asb ) / 2.0 ) )
